app/models/screenshot.rb

```diff
@@ -108,7 +108,14 @@ def generate_thumbnail(content)
 
   def fetch
     Rails.logger.debug("Fetching screenshot from #{source_url}")
-    content = open(source_url, "rb", :read_timeout => 6)
+    begin
+      content = open(source_url, "rb", :read_timeout => 6)
+    rescue Exception => e
```

Comment:
The exception is for example:

> Fetching screenshot from http://entangle-photo.org/appdata/en_US/screenshots/camera-manager.png
Exception: RuntimeError: redirection forbidden: http://entangle-photo.org/appdata/en_US/screenshots/camera-manager.png -> https://www.entangle-photo.org/appdata/en_US/screenshots/camera-manager.png

Not sure how to access this directly in Ruby. `RuntimeError` sounds a bit generic.